src/main/java/org/jenkinsci/plugins/microsoft/appservice/WebAppDeploymentRecorder.java

```diff
@@ -52,11 +52,11 @@
 import java.io.IOException;
 import java.util.Collections;
 
-public class AppServiceDeploymentRecorder extends Recorder implements SimpleBuildStep {
+public class WebAppDeploymentRecorder extends Recorder implements SimpleBuildStep {
```

Comment:
Let's also change the package name to start with com.microsoft.jenkins to follow our convention